functional_tests/environment.py

```diff
@@ -129,3 +149,19 @@ def after_scenario(context: Context, scenario: Scenario):
         context.playwright_context.tracing.stop_chunk()
 
     context.page.close()
+
+
+def before_tag(context: Context, tag: str):
+    """Handle tag-specific setup."""
+    if tag == "cognito_enabled":
+        # Apply Cognito test settings
+        settings = get_overridden_settings()
```

Comment:
Had another review and realised this is only used here, so perhaps do inline them. or rename to `get_auth_overridden_settings` or `get_cognito_overridden_settings`